src/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface ChromaticConfig {
+  delay?: number;
+  diffIncludeAntiAliasing?: boolean;
+  diffThreshold?: number;
+  disableAutoCapture?: boolean;
+  forcedColors?: string;
+  pauseAnimationAtEnd?: boolean;
+  prefersReducedMotion?: string;
```

Comment:
Do these work with the iframed snapshot?

Reply:
Oh I'm not sure, does that require some level of dom access?

Reply:
I'd assume so, but I haven't checked.